packages/mobile/src/screens/audio-screen/Panel.tsx

```diff
@@ -53,12 +55,19 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
     fontFamily: typography.fontByWeight.demiBold,
     lineHeight: spacing(4),
     color: palette.secondary,
-    backgroundColor: palette.background,
     borderWidth: 1,
     borderRadius: 12,
     borderColor: palette.backgroundSecondary,
     overflow: 'hidden'
   },
+  readyToClaimPill: {
+    backgroundColor: palette.background
+  },
+  newChallengeText: {
+    textShadowOffset: { width: 0, height: 1.4 },
+    textShadowRadius: 8,
```

Comment:
spacing?

Reply:
It's a one-off hard-coded value from figma that I don't think is supposed to be tied to the grid. But generally I'm with you 101%.